op-ufm/pkg/config/config.go

```diff
@@ -79,6 +80,27 @@ func New(file string) (*Config, error) {
 	if _, err := toml.DecodeFile(file, cfg); err != nil {
 		return nil, err
 	}
+
+	KMSKeyID := os.Getenv("KMS_KEY_ID")
+	KMSEndPoint := os.Getenv("KMS_ENDPOINT")
+	KMSRegion := os.Getenv("KMS_REGION")
+	for _, wallet := range cfg.Wallets {
```

Comment:
Seems like kind of odd behavior to set all configured wallets with the same kms info?  Maybe it's worth at least logging that an override is happening?

I haven't looked, but I'm assuming that the configuration isn't using something like urfavcli that has first class environment overrides?

Reply:
There is no `urfavcli`. It loads the whole configuration from the toml file. It just simplifies @rupertbg 's work.